main.py

```diff
@@ -66,7 +66,7 @@ def invoke_black_on_changed_files(args, changed_python_files):
     if not changed_python_files:
         return 0, ""
     with open("/tmp/changed_python_files.txt", "w") as f:
-        f.writelines(changed_python_files)
+        f.writelines([file_path + "\n" for file_path in changed_python_files])
```

Comment:
unfortunately `writelines` does not add a newline character after every line as you might expect: https://stackoverflow.com/questions/13730107/writelines-writes-lines-without-newline-just-fills-the-file